RELEASING.md

```diff
@@ -0,0 +1,79 @@
+# Releasing knative/pkg
+
+We release the components of Knative ever 6 weeks.  All of these components must
+be moved to the latest "release" of the knative/pkg shared library prior to each
+release, but likely this should happen incrementally over each milestone.
+
+## Release Process.
+
+### Step #1: Monday the week prior to each release.
+
+Early the week prior to the Knative release, each of the downstream repositories
+should stage a `[WIP]` Pull Request that advances the knative/pkg dependency to
+the latest commit:
+
+`Gopkg.toml` should look like:
+
+```toml
+[[override]]
+  name = "github.com/knative/pkg"
+  branch = "master"
+```
+
+Then the following is run:
+
+```shell
+dep ensure -update github.com/knative/pkg
+```
+
+If problems are found, they are addressed and the update is merged, and this
+process repeats until knative/pkg can be cleanly updated without any changes.
+
+> If by mid-week, we do not have a clean PR in each repository, the person
+> driving the update should escalate to avoid delaying the release.
+
+
+### Step #2: Friday the week prior to each release.
+
+A release branch is snapped on knative/pkg with the form `release-0.X` where `X`
```

Comment:
how does one "snap" a branch? it is automated?

Reply:
I usually do it through the github UI :)